README.md

```diff
@@ -4,7 +4,7 @@ Mobile Project Template
 ###appbuilder-mobile-template
 A project template for creating mobile applications designed for use with a Progress Data Service (OpenEdge or Rollbase).
 
-<a href="https://platform.telerik.com/#appbuilder/clone/https%3A%2F%2Fgithub.com%2FCloudDataObject%2Fappbuilder-mobile-template" target="_blank"><img src="http://docs.telerik.com/platform/appbuilder/sample-apps/images/try-in-appbuilder.png" alt="Try in AppBuilder" title="Try in AppBuilder" /></a>
+<a href="https://platform.telerik.com/#appbuilder/clone/https%3A%2F%2Fgithub.com%2FCloudDataObject%2Fappbuilder-mobile-template%2Ftree%2Fdev_4.2" target="_blank"><img src="http://docs.telerik.com/platform/appbuilder/sample-apps/images/try-in-appbuilder.png" alt="Try in AppBuilder" title="Try in AppBuilder" /></a>
```

Comment:
Maybe this should be returned to its original "href" before merge in master, because the way it is now, when the Pull Request gets merged,  users will be cloning from "dev_4.2" branch.